Vimeo-Native/VimeoNative.autopkg

```diff
@@ -15,7 +23,7 @@ nuget
       releaseNotes: "Initial Checkin";
```

Comment:
Should probably remove old "releaseNotes" (maybe was just going to happen on master when you rev the version?)